src/components/Sheet/ScriptureTable.tsx

```diff
@@ -1768,6 +1782,11 @@ export function ScriptureTable(props: IProps) {
   };
 
   const doPublish = async () => {
+    if (!firstBook) {
+      showMessage(t.setupBookFirst);
+      return;
+    }
+
```

Comment:
we talked about moving this to where it shows the are you sure message and showing this instead there